services/src/main/java/org/exoplatform/addons/gamification/service/configuration/RuleService.java

```diff
@@ -184,6 +185,27 @@ public RuleDTO findRuleByEventAndDomain (String ruleTitle, String domain) {
             throw(e);
         }
     }
+
+    /**
+     * Get all Rules from DB
+     * @return RuleDTO list
+     */
+    @ExoTransactional
+    public List<RuleDTO> getActiveRules() {
+        try {
+            //--- load actives Rules
+            List<RuleEntity> rules =  ruleDAO.getActiveRules();
+            if (rules != null) {
+                return ruleMapper.rulesToRuleDTOs(rules);
+            }else{
+                return Collections.emptyList();
+            }
+
+        } catch (Exception e) {
+            LOG.error("Error to find active rules",e);
+            throw(e);
+        }
```

Comment:
As ruleDAO.getActiveRules does not return null, and as persistence exception is managed in the dao layer, i think we could simplify this function : 
```suggestion
        List<RuleEntity> rules =  ruleDAO.getActiveRules();
        return ruleMapper.rulesToRuleDTOs(rules);
```